README.md

```diff
@@ -117,6 +117,24 @@ scoreapi$ black .
 
 <br/>
 
+## Use as a pip package
+
+### Linux/MacOS
+
+```bash
+$ python -m pip install scoreapi
+[...]
+Successfully installed scoreapi
+```
+### Windows
+```bash
+C:>  py -m pip install scoreapi
+[...]
+Successfully installed scoreapi
+```
+
```

Comment:
How do I use the package after installing it? What is the input format?

Reply:
The function we want to expose is `getScore` https://github.com/openfinanceafrica/scoreapi/blob/19984fbd601d88d1e67f3b52513849738271990e/src/score.py#L33

You can see how it's used by the API here: https://github.com/openfinanceafrica/scoreapi/blob/19984fbd601d88d1e67f3b52513849738271990e/src/app.py#L43

So the example will look similar to that with the exception of including what the import code should look like. As I mentioned before there might be a need to refactor the code a little bit since it was initially make for just API consumption.

I hope this helps. Let me know if you need any more clarification.

Reply:
Am having trouble implementing this.

Reply:
No worries. Could you let me know exactly where you’re stuck?

Reply:
> Could you also add a code example using the package (i.e. importing and using `getScore`)? Take a look at the API reference here: https://app.theneo.io/open-finance-africa/documentation_2/api_reference

This step

Reply:
That’s still too vague:) What have you tried so far? Also let  me know if there’s something about what I’m asking that I need to clarify.

Reply:
After installing the package, when I try to import it I get an error message 'ModuleNotFoundError'

Reply:
Also if you can get a similar project and then walk me through it I may be able to know what's happening

Reply:
I actually haven’t built a python package before so you probably know as much as I do:) Let’s pause this PR for now and try again in the future.

Reply:
Okay.